thejamesgore/Foxtons-Dashboard
# style: Update dashboard styling and refactor TotalRevenue and PropertyRefferals components

# Pull Request: Update dashboard styling and refactor revenue and referrals components 

## Description 🚀
This pull request updates the dashboard styling using modern CSS techniques, as well as refactoring the `TotalRevenue.tsx` and `PropertyRefferals.tsx` components for improved performance and maintainability.

These changes improve the overall quality of the application, making it more user-friendly and easier to maintain over time.

## Changes Made 👉
- Updated the dashboard styling using modern CSS techniques
- Refactored the `TotalRevenue.tsx` and `PropertyRefferals.tsx` components for improved performance and maintainability

## Reason for Changes 🔍
The changes were made to improve the dashboard's visual design and user experience, as well as to simplify the codebase and improve performance.

## How to Test
To test these changes, you can follow these steps:

- Clone this branch to your local machine
- Install the necessary dependencies with npm install
- Start the application with npm start
- Verify that the application starts without any errors
- Verify that the dashboard has been updated with the new styling and that the `TotalRevenue` and `PropertyRefferals` components are functioning as expected

## Checklist

## Related Issues
None.

## Other Notes
None.
